aws-s3/scripts-ccao-data-raw-us-east-1/spatial-corner_lot.R

```diff
@@ -0,0 +1,360 @@
+# Script to generate shapefiles with indicators for corner lots.
```

Comment:
This script has had almost no changes since https://github.com/ccao-data/data-architecture/pull/260. The only change I made was to alter the location that the output data gets written to, so I'll leave a comment on that line below to call it out.